assignment4.txt

```diff
@@ -0,0 +1,9 @@
+Aly Milich
+
+1. A string is collection of characters that can make up words or sentences. In order to read a string in C, strings have to be split into character arrays which break apart the string into individual chars.
```

Comment:
the main difference is that a string is a char array but with a null terminator as the last element!